packages/react-ui-core/src/Ratings/RatingBar.js

```diff
@@ -10,6 +10,7 @@ import Star from './Star'
 
 export default class RatingBar extends PureComponent {
   static propTypes = {
+    id: PropTypes.string.isRequired,
```

Comment:
Instead of making this required, how about we add something to generate one if an `id` doesn't exist.  Maybe something like this:
```

import { randomId } from '@rentpath/react-ui-utils'

get id() {
  return this.props.id || randomId()
}
```

Passing this down as `context` may fix the issues with `LinearGradient`.